sdk/monitor/monitor-opentelemetry-exporter/review/monitor-opentelemetry-exporter.api.md

```diff
@@ -20,6 +26,15 @@ export interface ApplicationInsightsClientOptionalParams extends coreClient.Serv
     host?: string;
 }
 
+// @public (undocumented)
+export class ApplicationInsightsSampler implements Sampler {
+    constructor(samplingRate?: number);
```

Comment:
Nit: Python and dot net uses "sampling_ratio" to represent [0,1] value, and sample_rate to represent 0-100% value.